docs/content/development/extensions-core/druid-lookups.md

```diff
@@ -32,7 +51,7 @@ The current implementation provides 2 type of poll cache, the first is onheap (u
 User can also implement a different lookup polling cache by implementing `PollingCacheFactory` and `PollingCache` interfaces. 
 
 #### Loading lookup
-Loading cache strategy will load the key\value pair upon request on the key it self, the general algorithm is load key if absent.
+Loading cache strategy will load the keyalue pair upon request on the key it self, the general algorithm is load key if absent.
```

Comment:
another inadvertent change maybe? though it probably should be 'key/value' instead of 'key\value'